### PR TITLE
Update func tests for size in gravatar ViewHelper

### PR DESCRIPTION
**Pull request by froemken**

### CODE REVIEW

**Review by linawolf — APPROVED** [review]
Thanks for providing more tests!!

File: Tests/Functional/ViewHelpers/GravatarViewHelperTest.php [hunk 1]
```diff
@@ -108,6 +108,57 @@ public function renderWillAddSizeToGravatarUri(): void
...
+        $view = GeneralUtility::makeInstance(StandaloneView::class);
+        $view->setTemplateSource(sprintf(
+            $this->getFluidTemplateSource(),
+            '<blog:gravatar emailAddress="foo@example.com" size="-12" />',
+        ));
+
+        $content = $view->render();
+
+        self::assertMatchesRegularExpression(
+            '/s=1/',
```
Comment (linawolf) [inline Tests/Functional/ViewHelpers/GravatarViewHelperTest.php:123]: This would also match s=10...

Comment (froemken) [reply Tests/Functional/ViewHelpers/GravatarViewHelperTest.php]: Hello @linawolf 

I have completely reworked the Functional Test

Stefan

**Review by linawolf — APPROVED** [review]

File: Tests/Functional/ViewHelpers/GravatarViewHelperTest.php (hunk 1 above)
Comment (linawolf) [reply Tests/Functional/ViewHelpers/GravatarViewHelperTest.php]: Much better

**Pull request merged**